src/api/ResponseBuilder.ts

```diff
@@ -0,0 +1,81 @@
+import Discord, { Client, Message, TextChannel } from "discord.js";
+
+const WARN_COLOR = "#e3bb39";
+const FEEDBAP_COLOR = "#4b94e3";
+const SUCCESS_COLOR = "#33bd51";
+
+/**
+ * Send a warning message to the target channel.
+ */
+export const sendWarn = (response: string, message: Message) => {
+	const embed = new Discord.MessageEmbed().setColor(WARN_COLOR).setDescription(`⚠ ${response}`);
+	message.channel.send(embed);
+};
+
+/**
+ * Send a success message to the target channel.
+ */
+export const sendSuccess = (response: string, message: Message) => {
+	const embed = new Discord.MessageEmbed().setColor(SUCCESS_COLOR).setDescription(`✅ ${response}`);
+	message.channel.send(embed);
+};
+
+export const logFeedback = (client: Client, message: Message) => {
+	if (message.content.length < 16) {
+		sendWarn("Please be a little more descriptive!", message);
+		return;
+	}
+
+	const embed = new Discord.MessageEmbed()
+		.setColor(FEEDBAP_COLOR)
+		.addField("New Feedback arrived!", message.content, false);
+
+	const feedbackGuild = client.guilds.cache.get(process.env.CHECK_GUILD!);
+
+	if (!feedbackGuild) {
+		throw new Error("Failed to access feedback guild");
+	}
+
+	const feedbackChannel = feedbackGuild.channels.cache.get(process.env.FEEDBACK_CHANNEL!);
+
+	// check if feedback channel exists
+	if (!feedbackChannel) {
+		throw new Error("Failed to access feedback channel");
+	}
+	// check if is text channel
+	if (!feedbackChannel.isText()) {
+		return console.log(`Channel ${feedbackChannel.id} is not a text channel!`);
+	}
+
+	console.log(
+		`Feedback received - Sending message to channel #${(feedbackChannel as TextChannel).name} (${feedbackChannel.id})`
```

Comment:
I don't believe you need to cast here, the compiler knows that you just checked if it's not a TextChannel. 
cc: @SkyezerFox can you confirm this